datasets/crows_pairs/crows_pairs.py

```diff
@@ -0,0 +1,106 @@
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""CrowS-Pairs: A Challenge Dataset for Measuring Social Biases in Masked Language Models"""
+
+from __future__ import absolute_import, division, print_function
+
+import csv
+import json
+
+import datasets
+
+
+_CITATION = """\
+@inproceedings{nangia2020crows,
+    title = "{CrowS-Pairs: A Challenge Dataset for Measuring Social Biases in Masked Language Models}",
+    author = "Nangia, Nikita  and
+      Vania, Clara  and
+      Bhalerao, Rasika  and
+      Bowman, Samuel R.",
+    booktitle = "Proceedings of the 2020 Conference on Empirical Methods in Natural Language Processing",
+    month = nov,
+    year = "2020",
+    address = "Online",
+    publisher = "Association for Computational Linguistics"
+}
+"""
+
+_DESCRIPTION = """\
+CrowS-Pairs, a challenge dataset for measuring the degree to which U.S. stereotypical biases present in the masked language models (MLMs).
+"""
+
+
+_URL = "https://raw.githubusercontent.com/nyu-mll/crows-pairs/master/data/crows_pairs_anonymized.csv"
+
+_BIAS_TYPES = [
+    "race-color",
+    "socioeconomic",
+    "gender",
+    "disability",
+    "nationality",
+    "sexual-orientation",
+    "physical-appearance",
+    "religion",
+    "age",
+]
+
+_STEREOTYPICAL_DIRECTIONS = ["stereo", "antistereo"]
+
+
+class CrowsPairs(datasets.GeneratorBasedBuilder):
+    "CrowS-Pairs: A Challenge Dataset for Measuring Social Biases in Masked Language Models"
+
+    VERSION = datasets.Version("1.0.0")
+    BUILDER_CONFIGS = [
+        datasets.BuilderConfig(
+            name="crows_pairs",
+            version=datasets.Version("1.0.0", ""),
+        ),
+    ]
+
+    def _info(self):
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=datasets.Features(
+                {
+                    "id": datasets.Value("int32"),
+                    "sent_more": datasets.Value("string"),
+                    "sent_less": datasets.Value("string"),
+                    "stereo_antistereo": datasets.ClassLabel(names=_STEREOTYPICAL_DIRECTIONS),
+                    "bias_type": datasets.ClassLabel(names=_BIAS_TYPES),
+                    "annotations": datasets.Sequence(datasets.Sequence(datasets.ClassLabel(names=_BIAS_TYPES))),
+                    "anon_writer": datasets.Value("string"),
+                    "anon_annotators": datasets.Sequence(datasets.Value("string")),
+                },
+            ),
+            supervised_keys=None,
+            citation=_CITATION,
+            homepage="https://github.com/nyu-mll/crows-pairs",
+        )
+
+    def _split_generators(self, dl_manager):
+        filepath = dl_manager.download(_URL)
+        return [
+            datasets.SplitGenerator(name=datasets.Split.TEST, gen_kwargs={"filepath": filepath}),
```

Comment:
The data is only provided for evaluation purpose, so just returning the `TEST` split.